dags/vz_cris_import.py

```diff
@@ -0,0 +1,39 @@
+import os
+import docker
+from datetime import datetime
+
+from airflow.decorators import dag, task
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.environ.get("ENVIRONMENT", 'development') # operating environment
+
+@dag(
+    dag_id="vz-cris-import",
+    description="A process which will import the VZ CRIS data into the database on a daily basis using data on the SFTP endpoint",
+    schedule="0 7 * * *",
+    start_date=datetime(2023, 6, 1),
+    catchup=False,
+    tags=["vision-zero", "cris"],
+    on_failure_callback=task_fail_slack_alert,
+)
+def cris_import():
+    @task()
+    def invoke_cris_import_via_docker():
+        client = docker.from_env()
+        docker_image = "atddocker/vz-cris-import:latest"
```

Comment:
will there be a need for `vz-cris-import:production` and `vz-cris-import:staging`, or will we only ever need to use the latest tag?

Reply:
We don't really have a good setup to run a staging server of this because we only have one SFTP endpoint. We could set one up, but this ETL has never really had a concept of it that I've been aware of.

Reply:
Thank you for that question Chia. I had a comment on here earlier that I removed, because I think I had missed the point of your question. Sorry about that!

I think we can get by without a different build for this in production and staging. I think that the same script and image works for both situations and is entirely based on if the secrets coming out of 1PW are under the `development` or `production` sections.